pages/post/[slug].js

```diff
@@ -33,6 +33,8 @@ export default function Post({ post, status }) {
   if (post) {
     post = post.attributes;
     var published_on = post.published_on.replace(",", "");
+    var published_time = new Date(post.publishedAt).toLocaleTimeString();
+    var authorName = post.author_id.data.attributes.username;
```

Comment:
Let's assign the author directly to the post in `setPostFields` method. So its easy for us to get it directly. Do same for image also